cumulus_library/template_sql/sql_utils.py

```diff
@@ -116,7 +116,7 @@ def _check_data_in_fields(
 
     with base_utils.get_progress_bar(transient=True) as progress:
         task = progress.add_task(
-            "Detecting available encounter codeableConcepts...",
+            "Detecting available codeableConcepts...",
```

Comment:
Unrelated, but just noticed this - seems like a copy and paste from encounter-specific checks to generic checks, but kept the encounter wording.